src/verify.c

```diff
@@ -788,7 +788,7 @@ int verify_main(int argc, char **argv)
 		ret = EXIT_FAILURE;
 	}
 
-	/* this concludes the critical section, after this point it's clean up time, the disk content is finished and final */
+/* this concludes the critical section, after this point it's clean up time, the disk content is finished and final */
```

Comment:
I remember seeing this clang-format behavior before, and I think I worked around it by placing the comment *after* the label. Then clang-format didn't adjust indentation.